search/lib/api/stac.js

```diff
@@ -7,13 +7,13 @@ const stacExtension = require('../stac/extension');
 const { assertValid, schemas } = require('../validator');
 const { logger, makeAsyncHandler } = require('../util');
 
-async function search (event, params) {
```

Comment:
I'll revert these style changes, they were made automatically by my IDE.

Reply:
@dylnclrk , no worries on this but definitely see if you can revert the style changes.  I think we are all using VS Code which is configured to use ESLint semistandard rules:

```
{
  "extends": "semistandard",
  "env": {
    "jest": true
  }
}
```
Are you using something other than VS Code?

Reply:
Nope, I'm on VSCode. I think I just had a `Prettier` extension turned on which was making its own style changes.